go.mod

```diff
@@ -59,6 +59,8 @@ require (
 	sigs.k8s.io/yaml v1.3.0
 )
 
+require github.com/celestiaorg/optimint v0.1.2-0.20220411081455-538130ca9168
```

Comment:
Tagged release?

Reply:
No. It's not yet merged in optimint yet, not to mention releasing. (See: `This PR is mostly for visibility`)